js/proposalGenerator.js

```diff
@@ -22,7 +22,7 @@ ProposalGenerator.prototype.validate = function() {
     }
     catch (e) {
 
-        switch(e.message) {
+        switch (e.message) {

```

Comment:
One day we might want to use name instead of message ? 
I know that I'm way more incline to e.name than e.message particularly if we have specific name. 
It will be way good for now tho :D